### ISSUE TITLE
Math does not render in tensordot doc

### ISSUE THREAD

**Issue by samuelstjean** [numpy/numpy#7822]
Seems like the math from the docstring are not rendered properly in latex when viewing online [1], making it harder to decipher the function usage. In the note section which reads
~~~
Notes

Three common use cases are:
axes = 0 : tensor product $aotimes b$ axes = 1 : tensor dot product $acdot b$ axes = 2 : (default) tensor double contraction $a:b$
~~~
The math is not rendered properly.
Probably missing a little something like double $$, I just honestly don't know how to fix it.

[1] https://docs.scipy.org/doc/numpy/reference/generated/numpy.tensordot.html#numpy.tensordot

**Comment by simongibbons** [comment]
This has been fixed already in #7340

**Comment by samuelstjean** [comment]
My bad, I had looked into the still opened issues, not the closed ones, so no need for this issue then.
